src/Events/Received/Plugin/DeviceDidConnectEvent.ts

```diff
@@ -19,7 +19,9 @@ export default class DeviceDidConnectEvent extends AbstractReceivedBaseEvent {
     return this.eventPayload.deviceInfo.type;
   }
 
-  public get typeName(): 'StreamDeck' | 'StreamDeckMini' | 'StreamDeckXL' | 'StreamDeckMobile' | 'CorsairGKeys' {
+  public get typeName(): keyof typeof DeviceType {
```

Comment:
I tried to simplify this typing a bit, but I am curious if it's just safe to use `string` here?

Reply:
Gotcha, I'll leave it as `keyof typeof` then with your blessing. It seems to work for autocomplete, at least with Webstorm.